content/api/vision/_index.md

```diff
@@ -17,7 +17,8 @@ The markers in the list have some useful attributes:
 - `is_wall_marker()` - returns whether or not the marker is a [wall marker](marker-ids/#wall-markers).
 - `is_token_marker()` - returns whether or not the marker is a [token marker](marker-ids/#token-markers).
 - `id` - returns the [id](marker-ids) of the marker.
-- `pixel_centre` - returns the location in pixels of the centre of the marker in the captured image.
+- `pixel_centre` - returns the [pixel](coordinates/#pixel-coordinates) co-ordinates of the centre of the marker in the captured image.
+- `pixel_corners` - returns a list of the corners of the markers in [pixel](coordinates/#pixel-coordinates) co-ordinates, clockwise with the first marker being the top left corner.
```

Comment:
Based on usage in the following bullets, we appear to have a style preference for not hyphenating "coordinates".